.github/workflows/oclif-release.yml

```diff
@@ -0,0 +1,64 @@
+name: Oclif Release
+on:
+  workflow_dispatch:
+    inputs:
+      home-brew-branch:
+        description: 'Run workflow in home brew branch'
+        default: main
+      tag-name:
+        description: 'Tag released recently'
+        required: true
+      formula:
+          description: 'Formula to release in homebrew'
+          default: twilio
+      pre-release:
+        description: 'Homebrew release type'
+        default: 'false'
+jobs:
+  oclif-release:
+    name: Publish for ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    outputs:
+      sha256: ${{ steps.sha256.outputs.sha256 }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: macos-latest
+            asset_name: ${{ github.event.inputs.formula }}-v${{ github.event.inputs.tag-name }}
+            command_name: npx oclif-dev pack
+            publish: homebrew
+    steps:
+      - uses: actions/checkout@v2
+      - run: npm cache clear & npm ci
```

Comment:
[Q] Is cache clear at each workflow run desired?

Reply:
+1